### PR TITLE
update golang lint

### PR DESCRIPTION
**Pull request by yxxhero**
Signed-off-by: yxxhero <aiopsclub@163.com>

update golang lint

add gci: ensure import order.

### CODE REVIEW

**Comment by yxxhero** [comment]
@mumoshu ping

File: test/e2e/template/helmfile/tmpl_test.go [hunk 1]
```diff
@@ -6,8 +6,9 @@ import (
 	"testing"
 	"text/template"
 
-	"github.com/helmfile/helmfile/pkg/tmpl"
 	"github.com/stretchr/testify/require"
+
+	"github.com/helmfile/helmfile/pkg/tmpl"
```
Comment (mumoshu) [inline test/e2e/template/helmfile/tmpl_test.go:11]: Out of curiosity, is there any way to auto-organize imports this way?

I thought the standard goimports doesn't support sectioning. I tend to use an internal tool for my non-opensource company projects. VS code seemed to group imports into two sections, one for standard libs and another for non-std libs but not exactly sure...

**Review by mumoshu — APPROVED** [review]
Cool! Thanks for your effort 👍

**Comment by mumoshu** [comment]
Fixed a conflict.

**Review by mumoshu — APPROVED** [review]

**Pull request merged**